### ISSUE TITLE
Getting low accuracy on Evaluation

### ISSUE THREAD

**Issue by Jayshree-2111** [zengqunzhao/EfficientFace#21]
Hello Team,
I am trying to evaluate the model on the Test Set of RAF-DB. But I am getting the low accuracy. 

$python main.py --evaluate --data "/home/aitg/Documents/EfficientFace/RAF/" --gpu "0"
Training time: 04-21 14:58
Test: [ 0/24]	Loss 2.2484 (2.2484)	Accuracy 10.156 (10.156)
Test: [10/24]	Loss 2.4601 (2.3581)	Accuracy  6.250 ( 8.452)
Test: [20/24]	Loss 2.5689 (2.4966)	Accuracy  9.375 ( 7.366)
 *** Accuracy 9.550  *** 

Would you please suggest me some solution on how to evaluate it after training it on the test set?

**Comment by zengqunzhao** [comment]
> Hello Team, I am trying to evaluate the model on the Test Set of RAF-DB. But I am getting the low accuracy.
> 
> $python main.py --evaluate --data "/home/aitg/Documents/EfficientFace/RAF/" --gpu "0" Training time: 04-21 14:58 Test: [ 0/24] Loss 2.2484 (2.2484) Accuracy 10.156 (10.156) Test: [10/24] Loss 2.4601 (2.3581) Accuracy 6.250 ( 8.452) Test: [20/24] Loss 2.5689 (2.4966) Accuracy 9.375 ( 7.366) *** Accuracy 9.550 ***
> 
> Would you please suggest me some solution on how to evaluate it after training it on the test set?

Hi, thanks for your attention.
Can you check if you load the right pretraining model? It's not possible to get such a low acc.

**Comment by Jayshree-2111** [comment]
Would you once specify how to evaluate the model, once it is trained?

**Comment by zengqunzhao** [comment]
> Would you once specify how to evaluate the model, once it is trained?

Just loading the pretraining mode for the corresponding dataset, see Pre-trained Models Nov. 12, 2021 Update

**Comment by Jayshree-2111** [comment]
I have downloaded the dataset and trained the model. Basically, I have followed the steps specified:
Step 1: download basic emotions dataset of [RAF-DB](http://www.whdeng.cn/raf/model1.html).
Step 2: download pre-trained model from [Google Drive](https://drive.google.com/file/d/1sRS8Vc96uWx_1BSi-y9uhc_dY7mSED6f/view?usp=sharing), and put it into ./checkpoint.

Now, will you please tell me in the main.py, where do we need to mention the path?
Also, please elaborate the Step 3.

**Comment by zengqunzhao** [comment]
> I have downloaded the dataset and trained the model. Basically, I have followed the steps specified: Step 1: download basic emotions dataset of [RAF-DB](http://www.whdeng.cn/raf/model1.html). Step 2: download pre-trained model from [Google Drive](https://drive.google.com/file/d/1sRS8Vc96uWx_1BSi-y9uhc_dY7mSED6f/view?usp=sharing), and put it into ./checkpoint.
> 
> Now, will you please tell me in the main.py, where do we need to mention the path? Also, please elaborate the Step 3.

The project has been updated, you can find the test part there.

Best,
Zengqun
